api/napi/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "slint-node",
   "version": "1.3.0",
-  "main": "index.js",
+  "main": "index.ts",
```

Comment:
I never used `esbuild` before but I can give it a try. I will give me an other to get a feeling if I can get it work in acceptable time otherwise I will wait for your help.